### ISSUE TITLE
[BUG] updateItem 更新后 icon 位置偏移

### ISSUE THREAD

**Issue by lilonghe** [antvis/G6#3348]
- [x] I have searched the [issues](https://github.com/antvis/g6/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/hardcore-frost-ef42v)

### Steps to reproduce
1. 使用 icon
2. 调用 updateItem

| Environment | Info |
|---|---|
| g6 | 4.5.0 |
| System | macOS 12 |
| Browser | Chrome 96 |

**Comment by lilonghe** [comment]
```
import G6 from "@antv/g6";
const item2 = {
  id: "node2"
};
const data = {
  nodes: [item2]
};
const graph = new G6.Graph({
  container: "root",
  width: 500,
  height: 500,
  defaultNode: {
    type: "circle",
    size: 60,
    icon: {
      show: true,
      width: 30,
      text: "\ue853",
      fontFamily: "iconfont",
      fontSize: 30
    }
  }
});
graph.read(data);

setTimeout(() => {
  graph.updateItem(item2.id, item2);
}, 2000);
```
<img width="86" alt="截屏2021-12-04 下午12 14 54" src="https://user-images.githubusercontent.com/12867278/144696586-a033b20c-1a1b-44cc-82e6-932566283165.png">

**Comment by Yanyan-Wang** [comment]
感谢报告，会尽快解决

**Comment by Yanyan-Wang** [comment]
fixed by v4.5.1
